doc/releases/migration-guide-3.7.rst

```diff
@@ -64,6 +64,9 @@ Kernel
 
 * The named struct ``z_arch_esf_t`` is now deprecated. Use ``struct arch_esf`` instead. (:github:`73593`)
 
+* The header file :zephyr_file:`include/zephyr/arch/arch_interface.h` has been moved from
```

Comment:
Please point to the PR which introduced the change with ```(:github:`64987`)```
As this is in the migration guide, does this mean users should do anything, and if so what would that be?

Reply:
Updated with how.